lib/SyTest/Synapse.pm

```diff
@@ -278,6 +307,13 @@ sub start
       if ( $self->{pusher} ) {
          @command = ( @command, "--pusher-config" => $pusher_config_path );
       }
+
+      if ( $self->{synchrotron} ) {
+         @command = ( @command,
```

Comment:
I'd be tempted by

    push @command, "--synchrotron-config", ...

and also fix up the other cases above it.